ibl_pipeline/behavior.py

```diff
@@ -343,20 +343,47 @@ class CompleteTrialSession(dj.Computed):
     # sessions that are complete with trial information and thus may be ingested
     -> acquisition.Session
     ---
-    trial_session_complete: bool              # whether the session is complete
+    stim_on_times_status: enum('Complete', 'Partial', 'Missing')
+    rep_num_status: enum('Complete', 'Missing')
+    included_status: enum('Complete', 'Missing')
     """
 
     required_datasets =  ["_ibl_trials.feedback_times.npy", "_ibl_trials.feedbackType.npy", \
-                            "_ibl_trials.intervals.npy", "_ibl_trials.repNum.npy", \
-                            "_ibl_trials.choice.npy", "_ibl_trials.response_times.npy", \
+                            "_ibl_trials.intervals.npy", "_ibl_trials.choice.npy", \
+                            "_ibl_trials.response_times.npy", \
                             "_ibl_trials.contrastLeft.npy", "_ibl_trials.contrastRight.npy", \
-                            "_ibl_trials.stimOn_times.npy", "_ibl_trials.included.npy",\
                             "_ibl_trials.probabilityLeft.npy"]
 
     def make(self, key):
         datasets = (data.FileRecord & key & 'repo_name LIKE "flatiron_%"' & {'exists': 1}).fetch('dataset_name')
-        key['trial_session_complete'] = bool(np.all([req_ds in datasets for req_ds in self.required_datasets]))
-        if key['trial_session_complete'] is True:
+        is_complete = bool(np.all([req_ds in datasets for req_ds in self.required_datasets]))
+        if is_complete is True:
```

Comment:
As you ensure `is_complete` is boolean above, `if is_complete is True:` is redundant and can simply be `if is_complete:`